src/components/NotificationBar/NotificationBar.tsx

```diff
@@ -125,40 +125,40 @@ const Wrapper = styled.div<{
     display: flex;
     gap: ${spacingByChar(0.5)};
     align-items: center;
-    justify-content: space-between;
     background-color: ${bgColor};
     padding: ${spacingByChar(0.75)};
     color: ${fgColor};
   `,
 )
-const ContentsWrapper = styled.div<{ themes: Theme }>(
-  ({ themes: { spacingByChar } }) => css`
-    display: flex;
-    align-items: center;
-    flex-grow: 1;
-    gap: ${spacingByChar(0.5)};
-
-    /* flexbox で ellipsis するために min-width をつけて幅の計算を発生させている */
-    min-width: 0;
-
-    .smarthr-ui-Icon {
-      flex-shrink: 0;
-    }
-  `,
-)
-
-const MessageWrapper = styled(Sidebar)`
+const IconLayout = styled.div`
+  & > svg {
```

Comment:
imo: 必要以上に詳細度が高く見えました。
```suggestion
  .smarthr-ui-Icon {
```

Reply:
`.smarthr-ui-Icon` は `Icon` コンポーネントのエスケープハッチで、これをハードコードで指定すると変更に弱くなってしまうので、それを緩和するためにこのようなセレクタ指定にしている感じです。

Reply:
`.smarthr-ui-Hoge` をエスケープハッチにこだわらなくてもいいのかな、という気はします。
詳細度が不要に高いのが気になるので、styled-components の機能を使うのはどうですか?
https://styled-components.com/docs/advanced#referring-to-other-components

Reply:
スタイル内のコンポーネント指定は対象が複数だとうまく行かないようでした。
flex ボックスにすることで内側の要素に line-height が発生しないようにすることで対応しました。 https://github.com/kufu/smarthr-ui/pull/2469/commits/01101b05a66d9d38de7a33915dd913a7027ad9d7

Reply:
ありがとうございます!

> 内側の要素に line-height が発生しないようにする

これコメントにしておくとよさそうです!

Reply:
コメント追加しました f04c06d00fbdc6aa40373c50307581b487ef496d